src/pages/patientView/PatientViewPage.tsx

```diff
@@ -78,6 +78,7 @@ import ResourcesTab, { RESOURCES_TAB_NAME } from './resources/ResourcesTab';
 import { MakeMobxView } from '../../shared/components/MobxView';
 import ResourceTab from '../../shared/components/resources/ResourceTab';
 import TimelineWrapper from './timeline2/TimelineWrapper';
+import ClinicalEventsTables from 'pages/patientView/timeline2/ClinicalEventsTables';
```

Comment:
Minor but I think this is better:

```suggestion
import ClinicalEventsTables from './timeline2/ClinicalEventsTables';
```

Reply:
i actually gonna leave these because this is how my IDE adds them automatically.  i will look to see if it can use relative instead